spec/classes/init_spec.rb

```diff
@@ -33,17 +33,30 @@
 
   osfamily_matrix = {
     'Debian-7' => {
-      :architecture           => 'x86_64',
-      :osfamily               => 'Debian',
-      :operatingsystemrelease => '7',
-      :ssh_version            => 'OpenSSH_6.0p1',
-      :ssh_version_numeric    => '6.0',
-      :ssh_packages           => ['openssh-server', 'openssh-client'],
-      :sshd_config_mode       => '0600',
-      :sshd_service_name      => 'ssh',
-      :sshd_service_hasstatus => true,
-      :sshd_config_fixture    => 'sshd_config_debian',
-      :ssh_config_fixture     => 'ssh_config_debian',
+      :architecture              => 'x86_64',
```

Comment:
the arrows should be aligned only in their block, ie from ``Debian-7` => {` to the closing `}`

Reply:
by changing this, it adds whitespace changes that could conflict with other PR's making those harder to merge without adding anything